src/components/table/helpers/mixin-tbody-row.js

```diff
@@ -374,5 +362,16 @@ export default {
       // Return the row(s)
       return $rows
     }
+  },
+  computed: {
+    getComputedFields() {
```

Comment:
Avoid using the word `get` in computed prop names, as it implies that it is a method not a computed prop.

perhaps for this one: `hasFieldDetails`